lib/puppetlabs_spec_helper/rake_tasks.rb

```diff
@@ -111,8 +111,13 @@
     warn 'No files for parallel_spec to run against'
   else
 
-    args = ['-t', 'rspec']
-    args.push('--').concat(ENV['CI_SPEC_OPTIONS'].strip.split).push('--') unless ENV['CI_SPEC_OPTIONS'].nil? || ENV['CI_SPEC_OPTIONS'].strip.empty?
+    args = ['--type', 'rspec']
+    additional_options = if ENV['CI_SPEC_OPTIONS'].nil? || ENV['CI_SPEC_OPTIONS'].strip.empty?
+                           ['--format', 'progress']
+                         else
+                           ENV['CI_SPEC_OPTIONS'].strip.split + ['--format', 'progress']
+                         end
```

Comment:
Isn't this equivalent?
```suggestion
    additional_options = ENV['CI_SPEC_OPTIONS'].to_s.strip.split + ['--format', 'progress']
```

Thought I'd suggest to alter the order so user provided options come last. I think rspec prefers the later options so users could still override the format:
```suggestion
    additional_options = ['--format', 'progress'] + ENV['CI_SPEC_OPTIONS'].to_s.strip.split
```

Reply:
yes, but `ENV['CI_SPEC_OPTIONS'].to_s.strip.split` will fail if `ENV` returns nil.

Reply:
`nil.to_s` returns an empty string (`""`)

Reply:
:thinking: let me test again